src/OpenTelemetry.Instrumentation.StackExchangeRedis/TracerProviderBuilderExtensions.cs

```diff
@@ -78,7 +78,7 @@ public static class TracerProviderBuilderExtensions
             TracerProviderBuilder builder,
             IConnectionMultiplexer connection,
             StackExchangeRedisCallsInstrumentationOptions options,
-            Action<StackExchangeRedisCallsInstrumentationOptions> configure = null)
+            Action<StackExchangeRedisCallsInstrumentationOptions> configure)
```

Comment:
@Kielek You I think should revert this change. This requires a user to always pass in an argument to the extension method.

I think this looks much better 
```csharp
.AddRedisInstrumentation(connection, options)
```

than this:
```csharp
.AddRedisInstrumentation(connection, options, null)
```

Reply:
This also exposes an issue with publicAPI not running for this project.

Reply:
@utpilla, it is private method `private static TracerProviderBuilder AddRedisInstrumentation`

All calls are in 
`dotnet
        public static TracerProviderBuilder AddRedisInstrumentation(
            this TracerProviderBuilder builder,
            IConnectionMultiplexer connection = null,
            Action<StackExchangeRedisCallsInstrumentationOptions> configure = null)
`

Public API is fine.

Reply:
My bad! Never mind 😂